moveit_core/python/tools/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 set(MOVEIT_LIB_NAME moveit_python_tools)
 
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
+
 add_library(${MOVEIT_LIB_NAME}
   src/pybind_rosmsg_typecasters.cpp
 )
 add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
-target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES})
+target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${PYTHON_LIBRARIES})
```

Comment:
Does python need to be added to the package.xml and to the dependencies listed in  for this?  Does python need to be added to the dependencies in the `catkin_package` in moveit_core?  Should the find_package be moved into the moveit_core CMakeLists.txt for this?

Reply:
I tried adding python, but cmake still couldn't find it... Do you have an idea how to fix this? For now, I just silence the find_package call and that works fine. 

Linking Python is needed on macOS. Alternatively I think we could add a flag to allow undefined symbols.

Reply:
You probably still need the `find_package` but you also probably need to add it as a dependency to package.xml so it can be installed by the system in order to find and link.

If you quiet it does that mean it will silently fail when it isn't installed?

Reply:
It is now listed in the `package.xml` and thus this comment can be resolved.